backend/src/test/java/com/db/backend/validation/FullNameValidatorTest.java

```diff
@@ -0,0 +1,39 @@
+package com.db.backend.validation;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+//import org.junit.Assert;
+//import org.junit.runner.RunWith;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+//import org.mockito.Mockito.mock;
+//import org.mockito.InjectMocks;
+//import org.mockito.Mock;
+//import org.mockito.runners.MockitoJUnitRunner;
```

Comment:
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
```
Remover comentários do código